appendix.rst

```diff
@@ -344,14 +426,21 @@ below with their respective functionality.
    mount SIF images with ``squashfuse`` in unprivileged user namespace
    workflows.
 
-#. **SINGULARITY_SIGNAL**: Specifies a signal sent to the instance.
+#. **SINGULARITY_SIGNAL**: Specifies the signal to send to an instance.
```

Comment:
```suggestion
#. **SINGULARITY_SIGNAL**: Specifies the signal to send to an instance when stopping it.
```

Reply:
Hmm - this is an interesting one, as you could send a non-kill/term signal to the instance which doesn't stop it.... but the command is called stop.

Reply:
Not that different from unix `kill`, when you think about it... (can also be used to send signals that don't "kill" the recipient process).

Reply:
That's my point really. It is unix kill. This means that `singularity stop` with a specified signal isn't always stopping the container.

Reply:
```
#. **SINGULARITY_SIGNAL**: Specifies the signal to send to an instance with
   ``singularity instance stop``.
```